core/src/main/java/com/predic8/membrane/core/transport/Transport.java

```diff
@@ -30,13 +30,18 @@
 
 public abstract class Transport {
 
-	protected Set<IPortChangeListener> menuListeners = new HashSet<IPortChangeListener>();
+	protected Set<IPortChangeListener> menuListeners = new HashSet<>();
 
-	private List<Interceptor> interceptors = new Vector<Interceptor>();
+	private List<Interceptor> interceptors = new Vector<>();
 	private Router router;
 	private boolean printStackTrace = false;
 	private boolean reverseDNS = true;
-	int concurrentConnectionLimitPerIp = 60;
+
+	/**
+	 * Number of allowed concurrent connections from one client ip
+	 * @default -1 No Limit
+	 */
+	private int concurrentConnectionLimitPerIp = -1;
```

Comment:
This Javadoc comment will *not* be picked up by the annotation processor, as the target is not a setter method (but a field).

Reply:
Ok. I'll move the comment to the setter.